config/locales/en.yml

```diff
@@ -51,6 +51,7 @@ en:
       index: "Users"
       new: "Personal details"
       check: "Check your answers"
+      show: "View a provider"
```

Comment:
```suggestion
      show: "View a user"
```

Reply:
Whoops - this is actually wrong. It should be the users name.

https://github.com/DFE-Digital/publish-teacher-training/pull/3166/commits/4942ce63b51f5d35b89a686fe219275775a9ad3f